example/src/screens/Meeting.tsx

```diff
@@ -232,7 +233,7 @@ const Meeting = ({
 
   return (
     <View style={styles.container}>
-      <ScrollView style={styles.scroll}>
+      <ScrollView style={styles.scroll} bounces={totalPeers > 4 ? true : false}>
```

Comment:
we can turn off the bounce altogether 
```suggestion
      <ScrollView style={styles.scroll} bounces={false}>
```